packages/react-dev-utils/webpackHotDevClient.js

```diff
@@ -63,8 +63,7 @@ var connection = new SockJS(
     protocol: window.location.protocol,
     hostname: window.location.hostname,
     port: window.location.port,
-    // Hardcoded in WebpackDevServer
-    pathname: '/sockjs-node',
+    pathname: process.env.WDS_SOCKET_PATH || '/sockjs-node',
```

Comment:
This seems to be repeated [here](https://github.com/facebook/create-react-app/pull/7750/files#diff-4f3337becb3b096953b3f341659a4bd3R92), could we find a way to share the logic?

Reply:
Letting it remain undefined within `react-scripts`. This allows someone to set it to an empty string (if they wanted to do that for whatever reason).

Within `react-dev-utils` we need to bake in a default. Repeated [here](https://github.com/facebook/create-react-app/pull/7750/files#diff-6736a5f1ee20761aaa6d0f4a09249e30R367).

I guess the way it stands now (after my change) is that `react-scripts` supports an empty string but `react-dev-utils` will force falsey to be the default of `'/sockjs-node'`. Internally, falsey is coerced to default in `webpack-dev-server`.

https://github.com/webpack/webpack-dev-server/blob/eab6acd2972e39c8b2d83f7a6614d40b8e14bf63/lib/Server.js#L104-L108

Reply:
What's the best way to share the `'/sockjs-node'` default value within `react-dev-utils`?

Reply:
I think this is fine for the moment... If someone changed one and not the other, it would clearly stop working.

Reply:
@heygrady Shouldn't the `setEditorHandler` function also be modified to check for the host and port? That fetch also goes to WDS...